ros2/nightly/nightly/Dockerfile

```diff
@@ -46,18 +57,7 @@ ENV ROSDISTRO_INDEX_URL https://raw.githubusercontent.com/osrf/docker_images/mas
 
 # install python packages
 RUN pip3 install -U \
-    argcomplete \
-    flake8 \
-    flake8-blind-except \
-    flake8-builtins \
-    flake8-class-newline \
-    flake8-comprehensions \
-    flake8-deprecated \
-    flake8-docstrings \
-    flake8-import-order \
-    flake8-quotes \
-    pytest-repeat \
-    pytest-rerunfailures
+    argcomplete
```

Comment:
Is there still a reason that argcomplete is soloed out?

Looks like its also available upstream:  
https://packages.ubuntu.com/jammy/python/python3-argcomplete

I couldn't find what the build farm does for this, given argcomplete probably isn't used by the buildfarm.

Reply:
For a long time the version of argcomplete was too low for several ros2cli and/or colcon features.
Maybe things have evolved since. Looks like in official documentation it switched to apt for that as well: https://github.com/ros2/ros2_documentation/pull/1275/files.

On the other hand if the list of pip packages to install is empty the template will stop installing pip or setuptools altogether that may have side effects so I'd rather not do this here and solely fix the flake8 issue in this PR